docker-compose-configuration/metaphactory-docker-compose/rds-local/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "2.2"
+services:
+  platform:
+    ports:
+      - "10214:8080"
+    depends_on:
+      - blazegraph
+    container_name: "${COMPOSE_PROJECT_NAME}-metaphacts-platform"
+    image: "${PLATFORM_IMAGE}"
+    restart: unless-stopped
+    mem_limit: 1g
+    mem_reservation: 1g
+    environment:
+      - PLATFORM_OPTS=-Dconfig.environment.sparqlEndpoint=http://${COMPOSE_PROJECT_NAME}-blazegraph:8080/blazegraph/sparql ${PLATFORM_OPTS}
+      - JAVA_OPTS=
+      - GENERATE_PLATFORM_PASSWORD=${GENERATE_PLATFORM_PASSWORD}
+    networks:
+      - default
+      - platform_network
+    volumes:
+      - platform-apps:/apps
+      - ${RDS_SCRIPT_ROOT_DIRECTORY}/rds-local-runtime:/runtime-data:rw
+      # Applications
+      # - ${RDS_SCRIPT_ROOT_DIRECTORY}/rds-local-apps/rds-demo:/apps/rds-demo
```

Comment:
these should go into a `docker-compose.override.yml` file (also add that in `.env`). This file should be unchanged from the one in the platform repo.

Reply:
also, why is `rds-demo` excluded here?

Reply:
the paths should be relative to the docker-compose file, i.e. `./rds-local-apps/rds-xxx`

Reply:
Relative Paths is not working for docker volumes, so we use namespace variable to set root directory.

Reply:
relative paths are working if you start them with `./subfolder`. See e.g. https://stackoverflow.com/questions/46907558/docker-compose-relative-paths-vs-docker-volume
We use this a lot in our docker-compose setup for metaphactory, CI, ...